packages/documentation/copy/pt/reference/Namespaces and Modules.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Namespaces e Módulos
+layout: docs
+permalink: /pt/docs/handbook/namespaces-and-modules.html
+oneline: Como organizar o código em TypeScript através de módulos ou namespaces
+translatable: true
+---
+
+Este post descreve as várias formas de organizar seu código usando módulos e namespaces no TypeScript.
+Abordaremos também alguns tópicos avançados sobre como usar namespaces e módulos, e algumas armadilhas comuns ao usá-los no TypeScrypt.
+
+Veja a documentação de [Módulos](/docs/handbook/modules.html) para mais informações sobre os módulos do ES.
```

Comment:
To fix this you can change the link like this:
```suggestion
Veja a documentação de [Módulos](/pt/docs/handbook/modules.html) para mais informações sobre os módulos do ES.
```
**NOTE:** The Portuguese version does not available yet. So IMO, I think it is better to don't change it yet and wait to see what is owners decision for it.

Reply:
No problem! I'll fix this one later, but I think it might be better to keep this link leading to the Portuguese version and wait for it to be ready before merging this content into the final version.

Maybe @khaosdoctor, @alvarocamillont, @danilofuchs or @orta can help with this.